compiler/backend/src/org/jetbrains/kotlin/codegen/state/KotlinTypeMapper.kt

```diff
@@ -899,12 +899,21 @@ class KotlinTypeMapper @JvmOverloads constructor(
                 writeParameter(sw, JvmMethodParameterKind.THIS, thisIfNeeded, f)
             }
 
+            for (contextReceiverParameter in f.contextReceiverParameters) {
+                writeParameter(sw, JvmMethodParameterKind.CONTEXT_RECEIVER, contextReceiverParameter.type, f)
+            }
+
+            val contextReceiversParameters = f.contextReceiverParameters
+            for (contextReceiverParameter in contextReceiversParameters) {
```

Comment:
Minor. Seems irrelevant for this commit

Reply:
Seems like a completely unnecessary change at all, thanks :)